label_sync/cluster/label_sync_deployment.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2017 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: label_sync
+  labels:
+    app: label_sync
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: label_sync
+    spec:
+      terminationGracePeriodSeconds: 180
+      containers:
+        - name: label_sync
+          image: gcr.io/k8s-prow/label_sync:0.100
+          imagePullPolicy: Always
+          # args:
+          # - -debug
```

Comment:
Did you mean to include this commented-out config?

Reply:
Comment removed